packages/recorder/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@wowarenalogs/recorder",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "author": "WoW Arena Logs",
+  "scripts": {
+    "build:recorder": "tsc",
+    "lint": "eslint . --max-warnings 0",
+    "lint:fix": "eslint . --fix"
+  },
+  "dependencies": {
+    "@ffmpeg-installer/ffmpeg": "^1.1.0",
+    "electron-store": "^8.1.0",
+    "ffmpeg-static": "^5.2.0",
+    "fluent-ffmpeg": "^2.1.2",
+    "fs-extra": "^11.1.1",
+    "obs-studio-node": "https://storage.googleapis.com/spires-log-files/obs-studio-node-0.3.21-win64-27.0.2.tar.gz",
```

Comment:
Reminder: post notes on how this is built somewhere, move storage location to WAL project on gcp